extensions/wikia/WallNotifications/WallNotifications.class.php

```diff
@@ -464,9 +473,28 @@ protected function sendEmail( User $watcher, array $data ) {
 		return $watcher->sendMail( $data['$MAIL_SUBJECT'], $text, $from, $replyTo, 'WallNotification', $html );
 	}
 
+	protected function getEmailExtensionController( $notification, $watcherName ) {
+		$controller = false;
+
+		if ( !empty( $notification->data->article_title_ns )
+			&& MWNamespace::getSubject( $notification->data->article_title_ns ) == NS_WIKIA_FORUM_BOARD
+			&& $notification->isMain()
+		) {
+			$controller = 'Email\Controller\Forum';
```

Comment:
I think Nirvana will let you call the controller with or without the ```Controller``` suffix, but we should probably be consistent.  It'd be good to add ```Controller``` here or remove it on L485,487,489